seirmo/figures.py

```diff
@@ -0,0 +1,61 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import plotly.graph_objects as go
+
+
+class IncidenceNumberPlot(object):
+    """
+
+    Creates incidence number plots for the app
+
+    """
+    def __init__(self):
+        super(IncidenceNumberPlot, self).__init__()
+
+        # Instantiate an empty plotly figure
+        self._fig = go.Figure()
+
+    def add_data(self, pandas_DataFrame, time_key='Time',
```

Comment:
python variables are always lower case, so `pandas_dataframe`, but they should also not be too long. Maybe just `data` is enough?